### PR TITLE
Topology sidebar Alerts panel: toggle alerts accordion on text click

### PR DESCRIPTION
**Pull request by alfonsomthd**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=2184984

### CODE REVIEW

File: packages/shared/alert/AlertsPanel.tsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ const AlertBadge: React.FC<AlertBadgeProps> = ({
           <StatusIconAndText title={alerts.length.toString()} icon={icon} />
         </Button>
       </Badge>
-      <span className="odf-alerts-panel__badge-text">
+      <span className="odf-alerts-panel__badge-text" onClick={onClick}>
```
Comment (bipuladh) [inline packages/shared/alert/AlertsPanel.tsx:73]: Can we make the whole thing one div and put the `onClick` listener there?

Comment (alfonsomthd) [reply packages/shared/alert/AlertsPanel.tsx:73]: I tested with a div and the view broke: with a `<span>` it works. Should we go for `span` ?

Comment (alfonsomthd) [reply packages/shared/alert/AlertsPanel.tsx]: > I tested with a div and the view broke: with a `<span>` it works. Should we go for `span` ?

@bipuladh replaced fragment with `span`.

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**